cetmix_tower_server/models/cx_tower_variable_value.py

```
@@ -357,70 +452,17 @@ def create(self, vals_list):
-                )
-            if check_global:
-                value_char = search_result.filtered("is_global").mapped("value_char")
-                result.update(
-                    {"global": value_char and value_char[0] if value_char else None}
-                )
-
-        return result
+        context = self.env.context.copy()
+        for key in CONTEXT_KEYS_TO_REMOVE:
+            context.pop(key, None)
+        return self.with_context(context)  # pylint: disable=context-overridden
```

Pylint (3.3.8)

[warning] Unknown option value for 'disable', expected a valid pylint message and got 'context-overridden'

(W0012)